#Custom message for joint_impedence_move function
string terminatingCondition
string rateNom
int32 tics
